modules/services/imapnotify.nix

```diff
@@ -19,9 +21,10 @@ let
         Unit = { Description = "imapnotify for ${name}"; };
 
         Service = {
-          ExecStart = "${pkgs.goimapnotify}/bin/goimapnotify -conf ${
-              genAccountConfig account
-            }";
+          ExecStart =
+            "${getExe cfg.package} -conf '${config.xdg.configHome}/imapnotify/${
```

Comment:
Won't this prevent automatic restart of the service when the account configuration changes?

Reply:
Had a closer look and yeah it doesn't look like this will work. To be able to restart the service on configuration change we must ensure that the unit file content change between the generations. I would propose to keep using the direct Nix store path here. You can still create the link in config home, if you want.

Reply:
Ah, please forgive me.  I've made #4212 though I can't test it right now.